src/PiWeb.Volume/VolumeMetadata.cs

```diff
@@ -96,57 +84,57 @@ public VolumeMetadata(
 	/// <summary>
 	/// The number of Voxels in X-Dimension.
 	/// </summary>
-	public ushort PositionX { get => _PositionX; init => _PositionX = value; }
+	public ushort PositionX { get; init; }
 
 	/// <summary>
 	/// The number of Voxels in Y-Dimension.
 	/// </summary>
-	public ushort PositionY { get => _PositionY; init => _PositionY = value; }
+	public ushort PositionY { get; init; }
 
 	/// <summary>
 	/// The number of Voxels in Z-Dimension.
 	/// </summary>
-	public ushort PositionZ { get => _PositionZ; init => _PositionZ = value; }
+	public ushort PositionZ { get; init; }
 
 	/// <summary>
 	/// The number of Voxels in X-Dimension.
 	/// </summary>
-	public ushort SizeX { get => _SizeX; init => _SizeX = value; }
+	public ushort SizeX { get; init; }
 
 	/// <summary>
 	/// The number of Voxels in Y-Dimension.
 	/// </summary>
-	public ushort SizeY { get => _SizeY; init => _SizeY = value; }
+	public ushort SizeY { get; init; }
 
 	/// <summary>
 	/// The number of Voxels in Z-Dimension.
 	/// </summary>
-	public ushort SizeZ { get => _SizeZ; init => _SizeZ = value; }
+	public ushort SizeZ { get; init; }
 
 	/// <summary>
 	/// The size of a Voxel in X-Dimension (mm).
 	/// </summary>
-	public double ResolutionX { get => _ResolutionX; init => _ResolutionX = value; }
+	public double ResolutionX { get; init; }
 
 	/// <summary>
 	/// The size of a Voxel in Y-Dimension (mm).
 	/// </summary>
-	public double ResolutionY { get => _ResolutionY; init => _ResolutionY = value; }
+	public double ResolutionY { get; init; }
 
 	/// <summary>
 	/// The size of a Voxel in Z-Dimension (mm).
 	/// </summary>
-	public double ResolutionZ { get => _ResolutionZ; init => _ResolutionZ = value; }
+	public double ResolutionZ { get; init; }
 
 	/// <summary>
 	/// Gets or sets the metadata.
 	/// </summary>
-	public ICollection<Property> Properties { get; } = new List<Property>();
+	public IReadOnlyList<Property> Properties { get; } = [];

```

Comment:
That's a breaking change. Just saying.

Reply:
I know, we will bump the major version and I think that this is (until now) a not so popular feature.